x-pack/plugins/infra/server/lib/domains/metadata_domain/metadata_domain.ts

```diff
@@ -48,17 +48,9 @@ export class InfraMetadataDomain {
 
 const pickMetadata = (buckets: InfraMetadataAggregationBucket[]): string[] => {
   if (buckets) {
-    const metadata = buckets
-      .map(module => {
-        if (module.names) {
-          return module.names.buckets.map(name => {
-            return `${module.key}.${name.key}`;
-          });
-        } else {
-          return [];
-        }
-      })
-      .reduce((a: string[], b: string[]) => a.concat(b), []);
+    const metadata = buckets.map(module => {
+      return module.key;
```

Comment:
Not sure fully understand this change. Is it because of the dataset? If yes, should this be dataset instead of module.key?

Reply:
`module` is an aggregation bucket, the value we are looking for is stored on the `key` attribute.

Reply:
It might make sense to rename the argument in the anonymous function passed to `map()` to `dataset` as this has indeed nothing to do with modules any more?